spyder/plugins/editor/tests/test_introspection.py

```diff
@@ -99,13 +100,16 @@ def test_space_completion(setup_editor):
     assert code_editor.toPlainText() == 'from numpy import\n'
 
 
-@pytest.mark.slow
+# @pytest.mark.slow
 def test_hide_widget_completion(setup_editor):
     """Validate hiding completion widget after a delimeter or operator."""
     editor, qtbot = setup_editor
     code_editor = editor.get_focus_widget()
     completion = code_editor.completion_widget
 
+    delimiters = ['(', ')', '[', ']', '{', '}', ',', ':', ';', '@', '=', '->',
```

Comment:
Question @steff456: Are this defined on some widget so that we can import the widget, instead of rewriting the delimiters here?

Otherwise it looks great :-p

Reply:
No, they are defined as a set inside keyPressEvent function in the code editor.

Reply:
Cool! Thanks @steff456 :-)